Makefile

```diff
@@ -0,0 +1,22 @@
+.DEFAULT_GOAL 	:= help
+
```

Comment:
@HarshCasper Please add `go test` commands for individual packages here

Reply:
Done!